daemon/daemon.go

```diff
@@ -76,21 +76,35 @@ func (d *Daemon) ListServices(ctx context.Context, namespace string) ([]v6.Contr
 	}
 
 	var services policy.ResourceMap
+	var noRepo bool
 	err = d.WithClone(ctx, func(checkout *git.Checkout) error {
 		var err error
 		services, err = d.Manifests.ServicesWithPolicies(checkout.ManifestDir())
 		return err
 	})
-	if err != nil {
+	switch {
```

Comment:
I like the use of the bare switch statements here and below - keeps all the references to each new bit of state easily within a screenful. Should be easier to read and follow than some of our bigger methods.